coordinator/src/services/verification/signature-verification.ts

```diff
@@ -0,0 +1,123 @@
+import {ElementCompact} from "xml-js"
+import {common, hl7V3} from "@models"
+import {writeXmlStringCanonicalized} from "../serialisation/xml"
+import {convertFragmentsToHashableFormat, extractFragments} from "../translation/request/signature"
+import {createParametersDigest} from "../translation/request"
+import crypto from "crypto"
+import {isTruthy} from "../translation/common"
+
+function comparePrescriptions(p1: common.Prescription, p2: common.Prescription): Array<string> {
```

Comment:
should we move this to a `compare-prescriptions.ts` file? the test below will match the file path then

Reply:
Done ✔️